release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -891,6 +891,13 @@ With this update, you can switch the BlueField-2 network device from data proces
 
 For more information, see xref:../networking/hardware_networks/switching-bf2-nic-dpu.adoc#switching-bf2-nic-dpu[Switching Bluefield-2 from DPU to NIC].
 
+[id="ocp-412-pod-level-bonding"]
+==== Pod-level bonding for secondary networks
+
+Pod-level bonding for secondary networks is generally available (GA) from {product-title} 4.10.30.
```

Comment:
@jeana-redhat I felt this was OK as the GA link is in the 4.10 column. At 4.12 GA pod-level bonding was GA because I did not amend the entry in the column for 4.12

Reply:
Might want to double-check with @kalexand-rh , but my thinking is we should treat them as unrelated. If this was GA at the initial release of 4.12, then you can just say that it's generally available and not worry about where it debuts in the older versions of OCP v4 :) You could probably also omit the link in the TP table update on line 2266.